### PR TITLE
added: login signup UI

### PR DESCRIPTION
**Pull request by rkapoor10**
`Note: click on the link in the preview to view the auth pages`

Also please give suggestions to improve lines 22 and 23 in login.jsx and signup.jsx

- folder structure
folder-structure

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
good to go

File: src/components/auth/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                    <form className="form-styles" action="submit">
+
+                        <input className="input-field fs-1r fw-semibold" type="email"
+                            placeholder="Enter Email/Mobile number"/>
+                        <input className="input-field fs-1r fw-semibold" type="password" placeholder="Enter Password"/>
+                        <span className="w-80"> <input className="privacy-terms"  type="checkbox" name="privacy-terms" id="privacy-terms"/> By continuing, you agree to Swift-UI <span className="baseteal fw-semibold">Terms
+                                of Use</span> and <span className="baseteal fw-semibold">Privacy Policy</span>.</span>
+
+                        <button
+                            className="btn btn-padding txt-s btn-outline secondary-cta-btn white bg-basepink w-80">LOGIN</button>
```
Comment (paarthbajaj) [inline src/components/auth/Login.jsx:35]: Just a suggestion, it is good to provide type to the button as it provides a clear definition.

**Review by Raghav888 — APPROVED** [review]
Good structure

**Review by paarthbajaj — APPROVED** [review]
You can merge, just have a look at my comment once.

File: src/components/auth/Login.jsx (hunk 1 above)
Comment (rkapoor10) [reply src/components/auth/Login.jsx:35]: sorry..but didn't get you

Comment (paarthbajaj) [reply src/components/auth/Login.jsx:35]: Read this once. https://dev.to/clairecodes/why-its-important-to-give-your-html-button-a-type-58k9#:~:text=When%20the%20type%20attribute%20is,behaves%20as%20a%20submit%20button.

**Pull request merged**